src/assets/style/LoginForm.css

```diff
@@ -0,0 +1,48 @@
+
+figure figcaption{
+    position:absolute;
+    top:50%;
+    left: 50%;
+    right: 0;
+    bottom: 0;
+    transform: translate(-50%,-25%); 
+  }
+  .letfSide_back-color{
+    background: linear-gradient(0deg, rgba(26, 40, 82, 0.8), rgba(26, 40, 82, 0.8)), linear-gradient(91.12deg, rgba(62, 71, 98, 0.372) 5.82%, rgba(10, 20, 49, 0) 99.07%);
+    box-shadow: 4px 0px 12px 2px rgba(0, 0, 0, 0.33);
+  }
+  .color-h1{
+    color: #1CA0E3;
```

Comment:
please work together with @sezeranoJchrisostome , to fix his PR as soon as possible and then bring the colors he difined into yours. Apply all colors where you have colors